ipmi-switches.yaml

```diff
@@ -0,0 +1,85 @@
+- name: "Gather required information"
+  hosts: ipmi_switches
+  gather_facts: false
+  tasks:
+
+    - name: "Calcuate rack and rackpos"
+      set_fact:
+        rack: "{{ parsed_address[-2] }}"
+        rackpos: "{{ parsed_address[-1] }}"
+        device_role: "ipmi_switches"
+      vars:
+        parsed_address: "{{ ansible_host | split('.') }}"
+
+    - name: "Create device role based on ansible group"
+      netbox.netbox.netbox_device_role:
+        netbox_url: "{{ netbox_url }}"
+        netbox_token: "{{ netbox_token }}"
+        data:
+          name: "{{ device_role }}"
+        state: present
+      delegate_to: 127.0.0.1
+
+    - name: "Create the device_type"
+      ignore_errors: true
+      netbox.netbox.netbox_device_type:
+        netbox_url: "{{ netbox_url }}"
+        netbox_token: "{{ netbox_token }}"
+        data:
+          manufacturer: "{{ ('brocade_icx' in group_names) | ternary('Brocade', 'IBM') }}"
+          model: "{{ group_names[0] }}"
```

Comment:
I think we want o fix this as well (and below on line 40).

Reply:
right, fixed it. I decided to actually put the switch model instead of just calling it `ibm`

Reply:
actually, wait.

Reply:
okay, it's ready for review now.